mods/persistence/game/objects/items/books/skill_book.dm

```diff
@@ -0,0 +1,48 @@
+#define MAX_TEXTBOOK_SKILLS   2
+#define TEXTBOOK_COOLDOWN     1 DAY
```

Comment:
Might be better to have it 20 hours, so that players who stayed up late one night and read at midnight don't have to always log in at midnight to read the book